astropy/coordinates/builtin_frames/icrs_cirs_transforms.py

```diff
@@ -20,7 +20,42 @@
 from .gcrs import GCRS
 from .cirs import CIRS
 from .hcrs import HCRS
-from .utils import get_jd12, aticq, atciqz, get_cip, prepare_earth_position_vel
+from .utils import get_jd12, aticq, atciqz, get_cip
+
+
+# utility function for transforms
+def prepare_earth_position_vel(time):
+    """
+    Get barycentric position and velocity, and heliocentric position of Earth
+
+    Parameters
+    -----------
+    time : `~astropy.time.Time`
+        time at which to calculate position and velocity of Earth
+
+    Returns
+    --------
+    earth_pv : `np.ndarray`
+        Barycentric position and velocity of Earth, in au and au/day
+    earth_helio : `np.ndarray`
+        Heliocentric position of Earth in au
+    """
+    # this goes here to avoid circular import errors
+    from ..solar_system import (get_body_barycentric, get_body_barycentric_posvel,
+                                solar_system_ephemeris)
+    # get barycentric position and velocity of earth
+    ephemeris = solar_system_ephemeris.get()
```

Comment:
there's also a problem with ```orbital_elements```, which ```solar_system``` imports, but which also imports ```ICRS```, ```GeocentricTrueEcliptic``` and ```SkyCoord```. It seems better to have the one inline import rather than three...

can tidy up the erfa import whilst we're here though